src_assets/common/assets/web/config.html

```diff
@@ -147,7 +147,7 @@ <h1 class="my-4">{{ $t('config.configuration') }}</h1>
           <div class="form-text">{{ $t('config.controller_desc') }}</div>
         </div>
 
-        <!-- Emulated Gamepad Type -->
+        <!-- Emulated Gamepad Type (Windows only) -->
```

Comment:
We can have one emulated gamepad block that covers Windows and Linux.

```html
        <!-- Emulated Gamepad Type -->
        <div class="mb-3" v-if="config.controller === 'enabled' && platform !== 'macos'">
          <label for="gamepad" class="form-label">{{ $t('config.gamepad') }}</label>
          <select id="gamepad" class="form-select" v-model="config.gamepad">
            <option value="auto">{{ $t('_common.auto') }}</option>
            <option value="ds4" v-if="platform === 'windows'">{{ $t('config.gamepad_ds4') }}</option>
            <option value="ds5" v-if="platform === 'linux'">{{ $t('config.gamepad_ds5') }}</option>
            <option value="switch" v-if="platform === 'linux'">{{ $t('config.gamepad_switch') }}</option>
            <option value="x360" v-if="platform === 'windows'">{{ $t('config.gamepad_x360') }}</option>
            <option value="xone" v-if="platform === 'linux'">{{ $t('config.gamepad_xone') }}</option>
          </select>
          <div class="form-text">{{ $t('config.gamepad_desc') }}</div>
        </div>
```

Reply:
Yeah, I've started with that and then refactored like it is now.  
I think it's cleaner with two separate blocks since they don't share any option, but I'd be happy to switch it back if you prefer it that way..

Reply:
I'd prefer the single block since it's less code to maintain, lol

Reply:
The config layout has been refactored. @Hazer can provide guidance if needed.